galaxy.yml

```diff
@@ -19,5 +19,15 @@ tags:
 - cisco
 version: 1.3.1
 build_ignore:
-- tests/output/*
+- 'tests'
 - '*tar.gz'
+- '*.DS_Store'
+- 'CHANGELOG.md'
```

Comment:
I would keep the CHANGELOG.md, it's nice for people to see it in the built collection they are downloading if they need to do it offline.

Reply:
Fixed.